### PR TITLE
chore(IT Wallet): [SIW-1773] Copy errors inside error screens

### PR DESCRIPTION
**Pull request by ale-mazz**
## Short description
This PR fixes some issues related to old copy inside our error screens and adjusts an error screen behavior

## List of changes proposed in this pull request
- Fixed some copy inside our error screens
- Removed the retry flow from an error screen [ItwIssuanceCredentialFailureScreen.tsx](https://github.com/pagopa/io-app/pull/6332/files#diff-d16d0ff009a0869ecf7150fad390276d2ae7d53616fc42396d6ffd94679ed8bc)
- Added retry flow inside an error screen [ItwIssuanceEidFailureScreen.tsx](https://github.com/pagopa/io-app/pull/6332/files#diff-6e81827f65b8e2bf66539bd0bc1af3eef64bd8a4bd685f383904160bc44cfb2b)

## How to test
Simulate an error while obtaining eID and while adding a credential. The new CTA while failing the matching identity should bring you to the identification mode screen

Preview:
IMAGE 2024-10-25 15:29:06

IMAGE 2024-10-25 15:29:23

### CODE REVIEW

File: ts/features/itwallet/issuance/screens/ItwIssuanceEidFailureScreen.tsx [hunk 1]
```diff
@@ -44,6 +44,13 @@ export const ItwIssuanceEidFailureScreen = () => {
     }
   };
 
+  const retryIssuance = (errorConfig?: KoState) => {
+    machineRef.send({ type: "retry" });
+    if (errorConfig) {
```
Comment (gispada) [inline ts/features/itwallet/issuance/screens/ItwIssuanceEidFailureScreen.tsx:49]: Do we really need this `errorConfig` and conditional logic if we never use it on line 113?

Comment (ale-mazz) [reply ts/features/itwallet/issuance/screens/ItwIssuanceEidFailureScreen.tsx]: After reviewing how the KO tracking had been handled, I thought I could refactor it by improving the tracking handling and adding some events that had not been tracked even though they were present in the document.
Fixed [here](https://github.com/pagopa/io-app/pull/6332/commits/d0948963c751dd54176cb6d8a34a244c7bec3efe)

Comment (ale-mazz) [reply ts/features/itwallet/issuance/screens/ItwIssuanceEidFailureScreen.tsx]: After reviewing how the KO tracking had been handled, I thought I would refactor it by improving the tracking handling and adding some events that were not tracked even though they were requested in the document.

**Review by gispada — APPROVED** [review]
LGTM

**Review by mastro993 — APPROVED** [review]
LGTM

**Pull request merged**